packages/itwinui-css/src/transfer-list/mixins.scss

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@mixin iui-transfer-list($area: null) {
```

Comment:
Can we move all this code into `transfer-list.scss`? No need to create mixins when we can just put the code directly under the classes

Reply:
Fixed